src/defs.qc

```diff
@@ -1696,5 +1696,7 @@ float	maybe_bprint_option_hack;
 .float	afk;
 entity	last_teleport;
 
+.float items2; // Cobalt, Engine will show rune on hud
```

Comment:
At one point, the models were related to the rune being picked up.  The problem become obvious, most people would focus on a particular model instead of picking up every rune.  The option is of course still there, but I run the server with random rune models and updating the hud with the current random model isn't at all useful.

I kind of wanted to use the hud info for something else, maybe as a timer or a locator.